### DISCUSSION TITLE
How to show tooltip without target but on specific element (by ref)

### DISCUSSION THREAD

**Discussion by 7iomka** [mantinedev/mantine#4725] [Q&A]
### What package has an issue

@mantine/core

### Describe the bug

I like how modals manager working now.
It is very useful when you don't have control for area where modal need to be rendered.
Unfortunately, with tooltip or with popover this is not a deal.
Both of it requires target as children.
I have filters on my marketplace app.
This is a long list of radio groups or checkbox group.
By design I need to show once the tooltip next to the last selected option after fetch request.
Note: option in this example is not a trigger, it is just an element and tooltip need to be calculated position and boundings around it.

How to do this?
Any ideas?

### What version of @mantine/hooks page do you have in package.json?

latest

### Do you know how to fix the issue

None

### Are you willing to participate in fixing this issue and create a pull request with the fix

None

**Comment by rtivital** [comment]
> By design I need to show once the tooltip next to the last selected option after fetch request.

Did not really get the issue. Why can't you render the last selected option inside Tooltip?

**Comment by 7iomka** [comment]
> > By design I need to show once the tooltip next to the last selected option after fetch request.
> 
> Did not really get the issue. Why can't you render the last selected option inside Tooltip?

My `FiltersList` looks like
```tsx

<Accordion
      value={openedFiltersKeys}
      onChange={setOpenedFiltersKeys}
      multiple
    >
      {filters.map((filter, idx) => (
        <Accordion.Item value={filter.key} key={`${filter.key}_${idx}`}>
          <Accordion.Control>{filter.label}</Accordion.Control>
          <Accordion.Panel>
            {filter.variant.type === 'checkbox' && (
              <FilterCheckbox
                options={filter.variant.options}
                value={filter.variant.value}
                onChange={(value) => onFilterChange({ key: filter.key, value })}
              />
            )}

            {filter.variant.type === 'radio' && (
              <FilterRadio
                options={filter.variant.options}
                value={filter.variant.value ?? filter.variant.defaultValue}
                onChange={(value) => onFilterChange({ key: filter.key, value })}
              />
            )}
            {filter.variant.type === 'color' && (
              <FilterColor
                options={filter.variant.options}
                value={filter.variant.value}
                onChange={(value) => onFilterChange({ key: filter.key, value })}
              />
            )}

            {filter.variant.type === 'range' && (
              <FilterRange
                defaultValue={filter.variant.defaultValue}
                value={filter.variant.value}
                onChange={(value) => onFilterChange({ key: filter.key, value })}
                className="py-10"
              />
            )}
          </Accordion.Panel>
        </Accordion.Item>
      ))}
    </Accordion>
```

Where, for example, FilterRadio is
```tsx
<Radio.Group className={className} value={selectedValue} onChange={onChange} py={10}>
      <Stack spacing={8}>
        {options.map(({ label, value, id }) => (
          <Radio
            value={value}
            label={label}
            key={value}
            id={id}
            className="capitalize"
            styles={{
              label: {
                paddingLeft: rem(8),
              },
            }}
          />
        ))}
      </Stack>
    </Radio.Group>
```

So, I have no idea, how and where to put Tooltip component when I'm in FiltersList component...
And I need this tooltip to be one-time - it should be shown at the last selected option, and this option not needed to be a constant target, because, how I previously said, I want to show this tooltip only after fetch request was finished (it triggers after user select an option)

**Comment by 7iomka** [comment]
Anyone?
